### ISSUE TITLE
Issues after upgrade react-native v0.71

### ISSUE THREAD

**Issue by Avinash-dev-code** [ammarahm-ed/react-native-mmkv-storage#319]
Describe the bug
Trying to upgrade to RN v0.71 but keep getting the following error

FAILURE: Build failed with an exception.

* Where:
Build file '/.../node_modules/react-native-mmkv-storage/android/build.gradle' line: 138

* What went wrong:
A problem occurred evaluating project ':react-native-mmkv-storage'.
> Expected directory '/.../node_modules/react-native-mmkv-storage/android/../../../node_modules/react-native/android' to contain exactly one file, however, it contains no files.
To Reproduce

upgrade to RN v0.71
Rebuild react-native android
See error
Expected behavior
react-native-mmkv-storage should continue working on the latest version of RN.

Screenshots

Platform Information:

OS: Android
React Native Version 0.71
Library Version 0.8.0
Gradle 7.5.1
Additional context

**Comment by Avinash-Gophygital** [comment]
@ammarahm-ed  Please give me some solution

**Comment by anatooly** [comment]
```
# The `/android` folder inside `react-native`

Starting from React Native 0.71, we're not shipping the `/android` folder inside the React Native NPM package
anymore due to sizing constraints on NPM. The Android artifacts are distributed via Maven Central.
You can read more about it in this RFC:
https://github.com/react-native-community/discussions-and-proposals/pull/508

If you're a library author and you're manipulating the React Native .aar files, to extract headers,
extract `.so` files or do anything with it, you're probably doing something wrong. React Native
0.71 ships with all the necessary logic to let you consume it transparently by just using:

implementation("com.facebook.react:react-android")
// or to keep backward compatibility with older versions of React Native:
implementation("com.facebook.react:react-native:+")

You should consider refactoring your library code not to unzip/manipulate the React Native .aar files.

This README.md file is kept in this folder as some libraries are checking the existence of the `/android` folder
and failing user builds if the folder is missing.
```

**Comment by anatooly** [comment]
> @ammarahm-ed Please give me some solution

Easy solution is change package & your code from react-native-mmkv-storage to react-native-mmkv

**Comment by Ahmedhamed77** [comment]
same problem here

**Comment by Ahmedhamed77** [comment]
> > @ammarahm-ed Please give me some solution
> 
> Easy solution is change package & your code from react-native-mmkv-storage to react-native-mmk

but this is another package :/

**Comment by anatooly** [comment]
> > > @ammarahm-ed Please give me some solution
> > 
> > 
> > Easy solution is change package & your code from react-native-mmkv-storage to react-native-mmk
> 
> but this is another package :/

yeap, but what different if you find more faster storage and minimum changes in your code?) https://github.com/mrousavy/react-native-mmkv/issues/100#issuecomment-886477361

30 min changes or waiting fix... week, month, year)))

**Comment by ammarahm-ed** [comment]
> > > > @ammarahm-ed Please give me some solution
> > > 
> > > 
> > > Easy solution is change package & your code from react-native-mmkv-storage to react-native-mmk
> > 
> > but this is another package :/
> 
> yeap, but what different if you find more faster storage and minimum changes in your code?) https://github.com/mrousavy/react-native-mmkv/issues/100#issuecomment-886477361
> 
> 30 min changes or waiting fix... week, month, year)))

Actually it's not my fault but react-native's that broke building the libraries in RN 0.71.0 completely. week, month,year is an exaggeration and you always send a PR to fix the issue. Performance is same.

**Comment by ammarahm-ed** [comment]
Fixed in 0.9.0.
